packages/syft-cli/setup.py

```diff
@@ -22,14 +23,14 @@
     author="OpenMined <info@openmined.org>",
     packages=find_packages(),
     install_requires=packages,
-    extras_require={"dev": dev_packages},
+    extras_require={"dev": dev_packages, "build": build_packages},
```

Comment:
`pip install -e packages/syft[dev]` should include pyinstaller as well right? With this we will have to install both `[dev]` and `[build]`

Reply:
Ah I see that it needs a fresh environment. So we can keep dev_packages i.e.
```
dev_packages = [
    "pytest",
] + build_packages
```

that way syft-cli[dev] will auto-include syft-cli[build]

Reply:
Done